src/Plugin/Field/FieldWidget/TextDateWidget.php

```diff
@@ -38,7 +38,8 @@ public function settingsForm(array $form, FormStateInterface $form_state) {
     $element = parent::settingsForm($form, $form_state);
     $element['date_format'] = [
       '#type' => 'textfield',
-      '#title' => $this->t('PHP DateTime Format String (http://php.net/manual/en/datetime.createfromformat.php).'),
+      '#title' => $this->t('PHP DateTime Format String.'),
+      '#description' => $this->t('See <a href="@phpdate" target="_blank">PHP DateTime Documentation</a> for details.', array('@phpdate' => 'http://php.net/manual/en/datetime.createfromformat.php')),
```

Comment:
You'll want to use `[ ]` instead of `array()` as I think you get code style violations now.

Reply:
I know, right. I've been caught doing it so many times in the past. But it's legal now! ;) https://www.drupal.org/docs/develop/standards/coding-standards#array